arches_component_lab/src/arches_component_lab/widgets/ResourceInstanceSelectWidget/components/ResourceInstanceSelectWidgetViewer.vue

```diff
@@ -1,16 +1,26 @@
 <script setup lang="ts">
 import arches from "arches";
 
-import type { ResourceInstanceReference } from "@/arches_component_lab/widgets/types.ts";
+import type { NodeData } from "@/arches_component_lab/types";
+import type { ResourceInstanceReference } from "@/arches_component_lab/widgets/types";
 
-const props = defineProps<{
-    value: ResourceInstanceReference | null | undefined;
+const { value } = defineProps<{
+    value: NodeData | null | undefined;
 }>();
 </script>
 <template>
-    <div :key="props.value?.resource_id">
-        <a :href="`${arches.urls.resource_editor}${props.value?.resource_id}`">
-            {{ props.value?.display_value }}
+    <div
+        :key="
+            (value?.interchange_value as ResourceInstanceReference).resource_id
+        "
+    >
+        <a
+            :href="`${arches.urls.resource_editor}${(value?.interchange_value as ResourceInstanceReference).resource_id}`"
+        >
+            {{
+                (value?.interchange_value as ResourceInstanceReference)
+                    .display_value
```

Comment:
I punted updating the viewers the "right" way until the removing interchange_value querysets ticket lands